modules/vmseries/variables.tf

```diff
@@ -277,4 +283,12 @@ variable "interfaces" {
     The `lb_backend_pool_id` cannot be `null` when `attach_to_lb_backend_pool` is set to `true`.
     EOF
   }
+  validation {
```

Comment:
You could also add a comment to this validation block following the pattern I mentioned.

Reply:
Added!

Reply:
Hey @horiagunica, I'm looking at it now and apparently you did not commit the change in the module variables file.